packages/formation-react/src/components/SortableTable/SortableTable.jsx

```diff
@@ -52,21 +98,39 @@ class SortableTable extends React.Component {
     );
   }
 
-  makeRow(item) {
-    const cells = this.props.fields.map(field => (
-      <td key={`${item.id}-${field.value}`}>{item[field.value]}</td>
-    ));
+  renderCell = (item, field) => {
+    const renderCustomCell = get(item, 'renderCustomCell');
+
+    // If the item has a `renderCustomCell` property, execute it.
+    if (renderCustomCell) {
+      return renderCustomCell(field);
```

Comment:
`return renderCustomCell ? renderCustomCell(field) : item[field.value];`

Reply:
I actually wrote it with the `if` statement so that it'll be easier to add on conditionals in the future if we want while still making the code readable :)